runtime/lua/vim/treesitter/_fold.lua

```diff
@@ -361,6 +386,20 @@ function M.foldexpr(lnum)
   return foldinfos[bufnr].levels[lnum] or '0'
 end
 
+function M.refresh()
```

Comment:
Does this need to be exposed? If it does can be prefix it with `_` so it doesn't need to be maintained under any API contract?

Reply:
Not really. Inlined the function to the OptionSet autocmd.